### ISSUE TITLE
admin level numbering, convention is 0=country, 1=first subnational etc.

### ISSUE THREAD

**Issue by andysouth** [covid19datahub/COVID19#146]
Thanks for the great work.

I initially expected your admin levels 1,2,3 to follow the convention that admin0=country, admin1=first subnational etc.  
e.g. https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country

Is there a reason for not following the convention ? Would be clearer to follow the convention if possible or at least provide clearer outline in the readme and documentation to avoid confusion.

**Comment by eguidotti** [comment]
Hello @andysouth, no there is no particular reason for not following the convention. Since R is 1-index I started with admin area 1 back in March. Now it is too late to change the naming convention as many people are already using it and this may cause troubles for the integrations that are already in place.

The convention is documented on the website [here](https://covid19datahub.io/articles/data.html). However I followed your suggestion and updated the README with some more info.

Hope this helps!
